src/Razor/Microsoft.NET.Sdk.Razor/src/build/netstandard2.0/Microsoft.NET.Sdk.Razor.Components.Wasm.targets

```diff
@@ -305,18 +300,22 @@ Copyright (c) .NET Foundation. All rights reserved.
     </Copy>
   </Target>
 
-  <Target Name="_CreateBlazorTrimmerRootDescriptorFiles" BeforeTargets="ILLink" AfterTargets="ComputeResolvedFilesToPublishList">
+  <Target Name="_BlazorWasmPrepareForLink" BeforeTargets="PrepareForILLink">
```

Comment:
Curious: where is the `PrepareForILLink` task defined?

Reply:
It was just added recently in the SDK:

https://github.com/dotnet/sdk/blob/e03d0f99bbe9daedec2c74297ae42d2a9279c5fe/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.ILLink.targets#L120-L131

See https://github.com/dotnet/sdk/pull/12116